EasyFinance.Domain/AccessControl/User.cs

```diff
@@ -63,5 +79,18 @@ public void SetTimezone(string timeZoneId)
             else
                 throw new ValidationException(nameof(this.TimeZoneId), ValidationMessages.InvalidTimeZone);
         }
+
+        public static bool IsValidEmail(string email)
+        {
+            try
+            {
+                var addr = new MailAddress(email);
+                return addr.Address != "";
+            }
+            catch
+            {
+                return false;
+            }
+        }
```

Comment:
```suggestion
```
Email is controlled by .Net Core 8 Authentication.

Because of that I suggest you don't create this validation.